pkg/collector/node_energy_collector.go

```diff
@@ -46,9 +48,13 @@ func (c *Collector) updatePlatformEnergy() {
 func (c *Collector) updateNodeComponentsEnergy() {
 	nodeComponentsEnergy := map[int]source.NodeComponentsEnergy{}
 	if components.IsSystemCollectionSupported() {
+		klog.V(5).Info("support components.IsSystemCollectionSupported")
```

Comment:
Can you update the message to a sentence instead of a code?
Otherwise it seems to be a debug message not a log.
For example:
`System energy collection is supported`

Reply:
done~